lib/util/state.js

```diff
@@ -338,8 +339,10 @@ function sortItineraries(type, direction, a, b, config) {
       else return b.duration - a.duration
     case 'COST':
       const configCosts = config.itinerary?.costs
-      const aTotal = getTotalFare(a, configCosts)
-      const bTotal = getTotalFare(b, configCosts)
+      // Sort an itinerary without fare information last (MAX_FARE is an arbitrarily high number)
+      const MAX_FARE = 999999.99 
```

Comment:
Use `Number.MAX_VALUE`.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/565/commits/dbd61ab2b015f36cba00f77eb716685e0c62b8e2